src/comprehensions.py

```diff
@@ -12,14 +12,23 @@
 
 y = []
 
-print (y)
+for x in range(5):
```

Comment:
The problems on line 15, 28, and 41 are asking you to use a list comprehension. Can you think of a way to make them like the one on line 52?